sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/http/HttpDownloadFileTask.java

```diff
@@ -41,9 +41,14 @@
     private static final int MAX_REDIRECTS = 6;
 
     /**
-     * The download progress will be reported every time this number of bytes is downloaded.
+     * The download progress will be reported after loading this number of bytes.
      */
-    private static final int UPDATE_PROGRESS_BYTES_COUNT = 128 * 1024;
+    private static final long UPDATE_PROGRESS_BYTES_THRESHOLD = 128 * 1024;
+
+    /**
+     * The download progress will be reported not more often than this number of milliseconds.
+     */
+    private static final long UPDATE_PROGRESS_TIME_THRESHOLD = 200;
```

Comment:
200ms is way too short for my example.
We really don't need that. I think download manager polls less often.

Reply:
> Let's increase the values to like 500ms and 512KiB